certbot/compat/filesystem.py

```diff
@@ -0,0 +1,29 @@
+"""Compat module to handle files security on Windows and Linux"""
+from __future__ import absolute_import
+
+import errno
+import os  # pylint: disable=os-module-forbidden
+
+
+def rename(src, dst):
```

Comment:
Actually one more comment. I've been thinking about our conversation in 3c and think that this function is more accurately called `replace` as it has the same behavior as that function on Python 3 and `os.rename` on Windows has different behavior.

(Bonus points for changing the implementation to match that by calling `replace` first if it exists and otherwise `rename`, but I'm less worried about that because the implementation isn't being added in this PR.)

Reply:
Done. With bonus point.